purrsoft/src/components/CustomCard/component.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { Card, CardContent, Box, Typography, useTheme } from '@mui/material';
+
+type CustomCardProps = {
+  width?: string | { xs: string; sm: string; md: string; lg: string };
+  height?: string | { xs: string; sm: string; md: string; lg: string };
+  align?: 'center' | 'flex-start' | 'flex-end';
+  justify?:
+    | 'center'
+    | 'space-between'
+    | 'space-around'
+    | 'flex-start'
+    | 'flex-end';
+  padding?: string;
+  title?: string;
+  subtitle?: string;
+  children: React.ReactNode;
+};
+
+export const CustomCard = ({
+  width = '300px',
+  height = '400px',
+  align = 'center',
+  justify = 'center',
+  title,
+  padding = '0px',
+  subtitle = '',
+  children,
+}: CustomCardProps) => {
+  const theme = useTheme();
```

Comment:
cred ca poti renunta la useTheme, pt ca in main.tsx oricum se foloseste ThemeProvider peste toate componentele din App si poti accesa direct culorile